nnbuilder/src/main/kotlin/org/hse/nnbuilder/nn/store/NeuralNetworkStored.java

```diff
@@ -13,7 +13,8 @@
 public final class NeuralNetworkStored {
 
     @Id
-    private Long id = new Random().nextLong();
+    private Long nnId = new Random().nextLong();
```

Comment:
Make sure you communicate this to all consumers if it results in column change.

Reply:
?
Actually I have changed it to SQL random